### ISSUE TITLE
Quest reward is showing false item level

### ISSUE THREAD

**Issue by braebrook** [Valhalla-Gaming/Tracker#8357]
**Describe the issue you're having**:
Just completed Moose on the Loose quest in Highmountain.  It said in the quest details that the reward was Medallion of Morashu and it was going to be item level 705, when I completed the quest it gave me the reward but it gave it to me as item level 680.

**Explain how you expect it work**:
I would think it would give it as the item level it would say.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Links to Wowhead, YouTube, etc**:

**Comment by braebrook** [comment]
Did the same thing when killing the bird when you signal the flare at her nest, supposed to get 699 Shoulders and they were 680 when accepting the reward.

**Comment by forstix** [comment]
another problem with the scaling issue i think

**Comment by Aival94** [comment]
Is a bug with the scaling as quest rewards scale with level but here seems to all stay at the same level version(98).

This item Scaling example(the numbers seems a little different than description here): [Medallion of Morashu](http://www.wowhead.com/item=141647/medallion-of-morashu&bonus=768)

**Comment by braebrook** [comment]
Yeah I think when I picked the quest up it said 705 was the reward but gave me 680. I'm really also hoping the damage issue gets fixed too because it's taking 1-2 minutes to kill a mob that has 401k health on those quests. The damage on the screen does not match what is being dealt to the mob.

Joe

**Comment by GM-Pyromanic** [comment]
@braebrook That is a visual issue, Once stat scaling issue is resolved it will not take players as long to kill mobs.

**Comment by lavalampe77** [comment]
That problem exists not only for quest rewards. In legion every loot item should have a chance to get a higher quality (item level). This feature should affect loot from PvP and PvE.
I have already reported this more then a year ago in warlords of draenor for rare mob drops but now in legion blizzard expanded that system for every loot.
https://github.com/Valhalla-Gaming/Tracker/issues/4283
